float32 value_a
uint16 value_b
uint16 value_c
uint16 window
bool ball_obs
bool RLE
bool kalman
